src/guides/v2.4/graphql/mutations/reset-password.md

```diff
@@ -0,0 +1,70 @@
+---
+group: graphql
+title: resetPassword mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `resetPassword` mutation resets customer password using a reset password token and the customer's email address. Use it to set a new password for the registered customer after calling the [requestPasswordResetEmail]({{page.baseurl}}/graphql/mutations/request-password-reset-email.html) mutation.
+
+## Syntax
+
+`mutation: {resetPassword(email: String!, resetPasswordToken: String!, newPassword: String!): Boolean}`
+
+## Example usage
+
+The following call sets a new customer password.
+
+**Request:**
+
+```graphql
+mutation {
+  resetPassword(
+    email: "roni_cost@example.com",
+    resetPasswordToken: "gh80pkjGdsPyiXc0sUUXswX1uGN7crUr",
+    newPassword: "new_password"
+  )
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "resetPassword": true
+  }
+}
+```
+
+## Input arguments
+
+The `resetPassword` mutation must contain the following arguments:
+
+Argument | Type | Description
+--- | --- | ---
+`email` | String! | Specifies the customer account that needs a password reset
+`resetPasswordToken` | String! | A runtime token
+`newPassword` | String! | The new password
+
+{:.bs-callout-info}
+Your new password must be six or more characters in length without spaces.
```

Comment:
Does the value of Customers > Customer Configuration > Password Options > Minimum Password Length determine this? Would it be better to say that the new password must meet the password policies set for the store?

Reply:
Good point! 

**fixed**